gsa/src/web/pages/help/about.js

```diff
@@ -116,7 +116,7 @@ const About = ({gmp}) => (
           </DivP>
           <DivP>
             {_('The GMP documentation is available ')}
-            <ProtocolDocLink title="here" />.
+            <ProtocolDocLink title={_('here')} />.
```

Comment:
The whole think will not be translatable but I also don't have a better solution

Reply:
The German translation is "Die GMP-Dokumentation finden Sie hier." and it's working perfectly on the about page. We need to deal with it again, once we offer more languages, but until then it's fine, I think,

Reply:
Yeah it will be fine for German and I really don't know a better or reliable solution currently. We'll leave it as it is.